### DISCUSSION TITLE
3D Printing Medical Hand splints

### DISCUSSION THREAD

**Discussion by mosin1457** [alicevision/AliceVision#995] [General]
Hi 

I know this may seem like an odd message, I currently am doing a masters in mechanical engineering, My project revolves around developing a method in 3D printing medical hand splints. One of the issues I have is obtaining an accurate 3D model of a patients forearm. Photogrammetry seems like a cheaper viable option however i need help in understanding this process, for example how does one obtain an accurate model, what are the conditions to achieve a good model. I hope someone can point me in the right direction. 

Thank you

**Comment by natowi** [comment]
- make yourself familiar with the basics of capturing images for photogrammetry
- you need to make sure the movement of the arm during capturing is being kept to a  minimum
- you could use a stand to support the arm in a rest position
- maybe digitizing a cast form could also be an option
- it is possible to capture images with one camera, but you would best automate the capturing process with  a camera that is moving around/along the fixated arm or design a tunnel (tubular camera rig)/cameras in a ring moving along to capture multiple images  the same time, minimizing the effects of movement
- you can find various similar rigs/photo booths/ maybe chaining a few https://www.openscan.eu/openscan-mini in a row?
- projected light patterns may be interesting, too https://github.com/alicevision/meshroom/wiki/Projected-Light-Patterns
- to be able to scale the model, place CCTAG markers in the scene with a known distance

**Reply by mosin1457** [reply]
Thank you so much!

**Comment by mosin1457** [comment]
Well here i am again, just providing an update and looking for some help. I have decided to attempt creating a 3D scan of a forearm using a iphone 11 camera. I will be testing the efficacy however the question i have is what is the best practice in order to do this, I have looked into building a rig however the project is not that advanced at this stage. i want to make a case that photogrammetry is a viable and cost effective option. So does anyone have any ideas on how which would be the best way to approach this, and how does one determine the quality of a scan?

**Reply by natowi** [reply]
> how does one determine the quality of a scan

This is done by comparing the reconstruction to a ground truth, for photogrammetry this is mostly done by comparing the reconstruction with a laser scan. Of course there are also other approaches, like evaluating the statistical parameters.
Best search for some papers on the topics to see some examples. 

To compare the results of different scans you could think about buying/borrowing a realistic silicon arm (prothesis/movie prop "fake arm" - _be aware you may see nsfw/graphical content when searching!_). [Here](https://www.dappercadaver.com/collections/fake-arms) is one supplier of fake polyfoam rubber arms, starting at 60$. It´s not cheap, but _you would have an arm that will be in the same position, regardless on how long you play around with your capturing equipment_. As the arm does not move you could compare the scan results and see which method works best, then you can try it with a real arm and compare the results.
